src/Lister/SubPageList.php

```diff
@@ -91,13 +91,15 @@ private function renderForTitle( Title $title, array $parameters ) {
 	/**
 	 * @param Title $title
 	 * @param int $limit
+	 * @param string $sortOrder
 	 * @param bool $includeRedirects
 	 *
 	 * @return Page
 	 * @throws LogicException
 	 */
-	private function getPageHierarchy( Title $title, $limit, $includeRedirects ) {
+	private function getPageHierarchy( Title $title, $sortOrder, $limit, $includeRedirects ) {
 		$this->subPageFinder->setLimit( $limit );
+		$this->subPageFinder->setSortOrder( $sortOrder );
```

Comment:
Inside `setSortOrder` I check that $sortOrder is 'asc' or 'desc'. I think this is secure enough.
What alternative should I use?